docs/reference-architecture/video-transcoding/diagrams/figure3.svg

```diff
@@ -0,0 +1,6173 @@
}.cls-76{mask:url(#mask-2);}.cls-77{mask:url(#mask-3);}.cls-78{mask:url(#mask-6);}.cls-79{mask:url(#mask-7);}.cls-80{mask:url(#mask-4);}.cls-81{mask:url(#mask-5);}.cls-82{mask:url(#mask-1);}.cls-9{fill:url(#linear-gradient-14);}.cls-9,.cls-10,.cls-36{opacity:.75;stroke:#ff5224;}.cls-10{fill:url(#linear-gradient-12);}.cls-11{fill:url(#linear-gradient-31);fill-opacity:.3;}.cls-11,.cls-13{stroke-dasharray:0 3;}.cls-86{filter:url(#luminosity-noclip-10);}.cls-87{filter:url(#luminosity-noclip-12);}.cls-88{filter:url(#luminosity-noclip-11);}.cls-12{stroke-dasharray:0 5.4981;stroke-width:1.2251px;}.cls-12,.cls-24,.cls-35,.cls-41,.cls-42{stroke:#393e44;}.cls-14{fill:url(#linear-gradient-55);}.cls-14,.cls-15,.cls-16,.cls-17,.cls-40{stroke-width:3.4912px;}.cls-15{fill:url(#linear-gradient-40);}.cls-16{fill:url(#linear-gradient-41);}.cls-17{fill:url(#linear-gradient-47);}.cls-18{fill:url(#linear-gradient-61);stroke-width:2.3925px;}.cls-83{mask:url(#mask-10);}.cls-84{mask:url(#mask-11);}.cls-19{fill:url(#linear-gradient-62);}.cls-20{fill:url(#linear-gradient-63);}.cls-21{fill:url(#linear-gradient-13);}.cls-22{fill:url(#linear-gradient-10);}.cls-85{mask:url(#mask);}.cls-25{fill:url(#linear-gradient);stroke:#fd8e3a;}.cls-26{fill:url(#linear-gradient-15);}.cls-27{fill:url(#linear-gradient-33);}.cls-28{fill:url(#linear-gradient-32);}.cls-29{fill:url(#linear-gradient-37);}.cls-30{fill:url(#linear-gradient-36);}.cls-31{fill:url(#linear-gradient-39);}.cls-32{fill:url(#linear-gradient-35);}.cls-33{fill:url(#linear-gradient-34);}.cls-34{fill:url(#linear-gradient-38);}.cls-36{fill:url(#linear-gradient-11);}.cls-37{fill:url(#linear-gradient-5);}.cls-38{fill:url(#linear-gradient-4);}.cls-39{fill:url(#linear-gradient-3);stroke-width:.7556px;}.cls-57{fill-rule:evenodd;}.cls-89{fill:url(#linear-gradient-59);}.cls-90{fill:url(#linear-gradient-58);}.cls-91{fill:url(#linear-gradient-53);}.cls-92{fill:url(#linear-gradient-54);}.cls-93{fill:url(#linear-gradient-51);}.cls-94{fill:url(#linear-gradient-52);}.cls-95{fill:url(#linear-gradient-50);}.cls-96{fill:url(#linear-gradient-43);}.cls-97{fill:url(#linear-gradient-42);}.cls-98{fill:url(#linear-gradient-45);}.cls-99{fill:url(#linear-gradient-46);}.cls-100{fill:url(#linear-gradient-44);}.cls-101{fill:url(#linear-gradient-49);}.cls-102{fill:url(#linear-gradient-48);}.cls-103{fill:url(#linear-gradient-18);}.cls-104{fill:url(#linear-gradient-22);}.cls-105{fill:url(#linear-gradient-20);}.cls-106{fill:url(#linear-gradient-21);}.cls-107{fill:url(#linear-gradient-23);}.cls-108{fill:url(#linear-gradient-19);}.cls-109{fill:url(#linear-gradient-16);}.cls-110{fill:url(#linear-gradient-17);}.cls-111{fill:url(#linear-gradient-27);}.cls-112{fill:url(#linear-gradient-24);}.cls-113{fill:url(#linear-gradient-26);}.cls-114{fill:url(#linear-gradient-25);}.cls-115{fill:url(#linear-gradient-28);}.cls-116{fill:url(#linear-gradient-29);}.cls-117{fill:url(#linear-gradient-30);}.cls-41{fill:url(#linear-gradient-60);}.cls-42{stroke-width:1.8977px;}</style>
```

Comment:
In the `<style>` tag, the linear gradients are all used in this form: `url(#linear-gradient-XX)`... those need to be prefixed to match the definitions below... e.g. `url(#svg3--linear-gradient)`